troposphere/codedeploy.py

```diff
@@ -177,10 +178,28 @@ class DeploymentGroup(AWSObject):
         'DeploymentGroupName': (basestring, False),
         'DeploymentStyle': (DeploymentStyle, False),
         'Ec2TagFilters': ([Ec2TagFilters], False),
+        'Ec2TagSet': ([Ec2TagFilters], False),
```

Comment:
Hey @mprince, I believe this requires a new Property type:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-codedeploy-deploymentgroup-ec2tagset.html